services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Persistence.kt

```diff
@@ -22,23 +21,30 @@ enum class SaveType {
 
 // Build-time plugins will open the class and add no-arg constructor for @Entity classes.
 
-@Entity @Table(name = "Hanke")
+@Entity @Table(name = "hanke")
 class HankeEntity (
     @Enumerated(EnumType.STRING)
     var saveType: SaveType? = null,
     var hankeTunnus: String? = null,
-    var startDate: Date? = null, // TODO: possibly ZonedDateTime, but we don't need the time or timezone...
-    var endDate: Date? = null, // TODO: possibly ZonedDateTime, but we don't need the time or timezone...
-    var owner: String? = null,
-    var phase: String? = null, // TODO: convert to enum, once known, and @Enumerated(EnumType.STRING)
-    var isYKTHanke: Boolean? = false,
-    var createdAt: Timestamp? = Timestamp.valueOf(LocalDateTime.now()),
-    var modifiedAt: Timestamp? = Timestamp.valueOf(LocalDateTime.now()),
+    var nimi: String? = null,
+    var kuvaus: String? = null,
+    var alkuPvm: LocalDate? = null, // NOTE: stored and handled in UTC, not in "local" time
```

Comment:
Aa, täällä onkin LocalDate - mutta mitä tuo NOTE tarkoittaa? Onko se huomio, että pitää ottaa huomioon tuo käsittely erikseen vai tarkoittaako se sitä, että JPA/Hibernate huolehtii että LocalDate menee kantaan UTC-ajassa?

Reply:
Se on muistutus tulevalle lukijalle että kannan tiedon on tarkoitus olla UTC vaikka siellä ei aikavyöhykettä olekaan, _ja_ vinkki siitä, että LocalDate, "Local"-sanasta huolimatta, ei ole mitenkään "paikallista" aikaa. (Sen nimi olisi ollut parempi "ZonelessDate" tai vastaava, mutta Javahan tunnetusti nimeää asioista hiukan sinne päin.)